src/app/apiAndObjects/api/intervention/getInterventioData.ts

```diff
@@ -0,0 +1,25 @@
+/* tslint:disable: no-string-literal */
```

Comment:
we dont use tslint any more - were you getting errors in vscode from the extension which needed this line?

Reply:
Removed it in current working branch